.travis.yml

```diff
@@ -2,33 +2,53 @@ language: python
 python: 3.7
 sudo: true
 dist: bionic
+
+env:
+  global:
+    - DOCKER_CACHE_IMAGE=athenian/api
+    - GOOGLE_DOCKER_IMAGE=gcr.io/athenian-1/api
```

Comment:
I don't feel strongly about the original tag name. How about having the same everywhere so that there is no need for retagging.

Reply:
we push the image with tag `gcr.io/athenian-1/api:branch-commit(7 symbols)` so we will retag anyway

Reply:
Anyway, there will be less code and simpler scripts.

Reply:
Done